components/ImprovedSearch/CustomClearRefinement.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button } from "@equinor/eds-core-react"
+import { tokens } from "@equinor/eds-tokens"
+import React from "react"
+import { useClearRefinements, UseClearRefinementsProps } from "react-instantsearch-hooks-web"
+import { useIntl } from "react-intl"
+import styled from "styled-components"
+
+const ButtonNew = styled(Button)`
+  color: ${tokens.colors.text.static_icons__default.hex};
+  background: ${tokens.colors.infographic.primary__moss_green_13.hex};
+  border-width: 0;
+  font-weight: 14pt;
+  padding-left: 8.15rem;
```

Comment:
You can't do this to position the button. 😄  If you tab through it or inspect it, you will notice that it doesn't look good. I would suggest to rather use `justify-content: space-between` on the header flex box instead
![Skjermbilde 2023-02-16 kl  10 00 38](https://user-images.githubusercontent.com/1568252/219319788-f6e30e2c-b545-45de-8029-f3f551dce074.png)
![Skjermbilde 2023-02-16 kl  10 00 55](https://user-images.githubusercontent.com/1568252/219319820-41aacaf4-7c19-46e6-8bf5-496221590e93.png)


![Skjermbilde 2023-02-16 kl  10 01 40](https://user-images.githubusercontent.com/1568252/219319859-bc0f2d23-92d2-4178-96b4-09af9bc29c60.png)